mslib/msui/topview.py

```diff
@@ -362,7 +359,7 @@ def openTool(self, index):
                                                             mscolab_server_url=self.mscolab_server_url,
                                                             token=self.token)
 
-                self.mainwindow_signal_logout_mscolab.connect(lambda: self.signal_logout_mscolab.emit())
+                self.mainwindow_signal_logout_mscolab.connect(self.signal_logout_mscolab.emit)
```

Comment:
as discussed in #2310 (commits) this was the culprit

Reply:
Could you please file an issue to change this type of lambda everywhere? I've seen it in other places as well and the construct is generally problematic.

Reply:
a follow up: https://github.com/Open-MSS/MSS/issues/2320